ReleaseNotesGenerator/ReleaseNotesGenerator/NuGet-6.4.md

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
Was this included by accident? This is the 6.4 output. 😊

Reply:
yep, deleted it :D